index.html

```diff
@@ -8,9 +8,9 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <div class="container" style="background-image:url('/home/ignatius/Documents/Microverse Projects/Portfolio/images/background-image.png');">
+    <div class="container">
         <div class="toolbar">
-            <p class="heading-1" style="color:blue ;">My logo</p>
+            <p class="heading">My logo</p>
         </div>
         <div>
             <h2 class="greetings">I am Ignatius Cheruiyot</h2>
```

Comment:
- [ ] Kindly as suggested by the previous reviewer, I think it would be better to replace this ```<h2>``` tag to a  ```<h1>``` tag as this is the main title of your web site, and this also will help in the search engine optimisation 👍